lib/hmmlearn/hmm.py

```diff
@@ -164,12 +169,13 @@ def _check(self):
         super()._check()
 
         self.emissionprob_ = np.atleast_2d(self.emissionprob_)
-        n_features = getattr(self, "n_features", self.emissionprob_.shape[1])
-        if self.emissionprob_.shape != (self.n_components, n_features):
+        if not self.n_features:
```

Comment:
`if self.n_features is None` because explicitly passing n_features=0 seems more like a programming error which should raise rather than something that should be silently overwritten.

Reply:
:+1: